pkg/login/social/generic_oauth.go

```diff
@@ -224,7 +251,12 @@ func (s *SocialGenericOAuth) UserInfo(client *http.Client, token *oauth2.Token)
 	return userInfo, nil
 }
 
-func (s *SocialGenericOAuth) extractToken(data *UserInfoJson, token *oauth2.Token) bool {
+func (s *SocialGenericOAuth) extractToken(userInfoResp []byte, token *oauth2.Token) bool {
+	var data UserInfoJson
+	if err := json.Unmarshal(userInfoResp, &data); err != nil {
+		return false
+	}
```

Comment:
`userInfoResp` will always be empty here, so there is no reason to try to unmarshal it.